doc/release-notes-0.13.md

```diff
@@ -0,0 +1,55 @@
+# 0.13 is not yet released! These are pre-release notes for features coming in 0.13. Some of these features may be released in 0.12 point releases
```

Comment:
Before 0.13 goes out I want to add a few endpoints that are aliases for existing functionality but clean up swagger/auto-gen.